stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -270,9 +340,39 @@ impl NakamotoBlockHeader {
 impl NakamotoBlock {
     /// Did the stacks tenure change on this nakamoto block? i.e., does this block
     ///  include a TenureChange transaction?
-    pub fn tenure_changed(&self) -> bool {
-        // TODO: when tenure change txs are implemented, this must be updated
-        true
+    pub fn tenure_changed(&self, parent: &StacksBlockId) -> bool {
```

Comment:
I think it might make more sense to return the _list_ of `TenureChange` transactions here.  The tenure can change multiple times in a block, since it has to contain all _missing_ `TenureChange` transactions as well.

Reply:
I think that will be necessary once the `TenureChange` transactions are actually getting validated, but right now, all the stacks block processor cares about is whether or not a tenure change occurred.